build.gradle

```diff
@@ -155,9 +155,13 @@ jar {
 }
 
 test {
-    // enable TestNG support (default is JUnit)
+    String CI = "$System.env.CI"
     useTestNG{
-        excludeGroups 'cloud', 'bucket'
+        if ( CI == "true" &&  "$System.env.CLOUD" =="true"){
```

Comment:
Can we make it so that to turn on cloud/bucket tests, all you have to do is set CLOUD to true?